### PR TITLE
added JS3-WK2 Homework

### PR DESCRIPTION
**Pull request by STEP0HEN**

### CODE REVIEW

File: javascript/javascript3/week2/JS3-wK2-Homework/visual-Promise/main.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                            {
+                                x: 400, y: 20
+                            })
+                    })
+                        .then(() => {
+                            console.log('Green lement has been moved');
+                        });
+                    });
+                    };
+   translateOnebyOne();
```
Comment (nmoskaleva) [inline javascript/javascript3/week2/JS3-wK2-Homework/visual-Promise/main.js]: The code works, but is extremely hard to read bc of improper indentation.

File: javascript/javascript3/week2/JS3-wK2-Homework/visual-Promise/main.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        const blueCircleTarget = moveElement(blueBox, 
+            {
+                x: 400, y: 300
+            });
+            const greenCircleTarget = moveElement(greenBox, 
+                {
+                    x: 400, y: 20
+                });
+    return Promise.all([redCircleTarget,blueCircleTarget,greenCircleTarget])
+    .then(()=> console.log("elements moved to target"))
```
Comment (nmoskaleva) [inline javascript/javascript3/week2/JS3-wK2-Homework/visual-Promise/main.js]: you could also move `.then` outside of the function.
`translateAllAtOnce().then(()=> console.log("elements moved to target"))`

**Review by nmoskaleva — COMMENTED** [review]
Well done on promises, but please pay attention to indentation. There are extensions that can help you with this, you can google "code formatter extension". If you're using VS Code, check out Prettier https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

**Comment by STEP0HEN** [comment]
Thank you for the review. I've implemented the changes.

**Pull request merged**